Code/Framework/AzToolsFramework/Tests/EditorViewportOnlyDrawHelpersForSelectedEntityTests.cpp

```diff
@@ -0,0 +1,186 @@
+/*
```

Comment:
update: I maybe wouldn't name the file the name of the test as it might be useful to add other related tests in here in future, so perhaps we could rename this (with `git mv <old-name> <new-name>`) to something like `EditorViewportHelpersTests`?